npm-frontend/src/CreateNewEvent.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const ChooseDefaultButton = styled.button`
+  background-color: limegreen;
+  color: white;
+  height: 150px;
+  width: 230px;
+  font-size: 40px;
+  border-radius: 13px;
+  cursor: pointer;
+  margin: 35px;
+  border-color: #272727;
+  font-weight: bold;
+  box-shadow: inset 0px 0px 30px #000000, 13px 12px 20px -5px black;
+  transition: width .5s, height .5s, font-size .5s, background-color .5s;
+
+  &:hover {
+    height: 170px;
+    width: 250px;
+    font-size: 40px;
```

Comment:
change the font size here to be bigger so the text grows at the same rate as the box